downstream/archive/archived-modules/platform/con-inventory-introduction.adoc

```diff
@@ -61,6 +61,9 @@ registry_password='<registry password>'
 
 The first part of the inventory file specifies the hosts or groups that Ansible can work with. 
 
-[NOTE]
+[WARNING]
```

Comment:
Just curious about this being labeled as a warning. See the RH Supplementary Guide about the types of admonitions [here](https://redhat-documentation.github.io/supplementary-style-guide/#admonitions). Warnings, in particular, are described as "Information about potential system damage, data loss, or a support-related issue if the user disregards this admonition." Should it remain a note?

Reply:
Yes, this would be a note. Thank you for the information. I know it's hard to be a stickler about new information, but I appreciate it always.,